eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -749,6 +749,33 @@
           "code" : "java.method.removed",
           "old" : "method com.azure.resourcemanager.containerservice.models.ManagedClusterNatGatewayProfile com.azure.resourcemanager.containerservice.models.ManagedClusterNatGatewayProfile::withEffectiveOutboundIPs(java.util.List<com.azure.resourcemanager.containerservice.models.ResourceReference>)",
           "justification": "The effectiveOutboundIPs was never a client-side modifiable property."
+        },
+        {
+          "regex": true,
+          "ignore": true,
+          "code": "java.class.nowFinal",
+          "new": "class com\\.azure\\.resourcemanager\\.eventgrid\\.models\\.(DeadLetterWithResourceIdentity|DeliveryWithResourceIdentity|EventSubscriptionIdentity)",
+          "justification": "Class is now final."
```

Comment:
User should not be able to extend these classes.

Reply:
fixed by [ce67d75](https://github.com/Azure/azure-sdk-for-java/pull/44748/commits/ce67d753263c7a158253671f5c5a50126603de6e)